src/Microsoft.IdentityModel.JsonWebTokens/Json/JsonWebToken.PayloadClaimSet.cs

```diff
@@ -14,12 +14,12 @@ public partial class JsonWebToken
     {
         internal JsonClaimSet CreatePayloadClaimSet(byte[] bytes, int length)
         {
-            return CreatePayloadClaimSet(bytes.AsSpan(0, length));
+            return CreatePayloadClaimSet(bytes.AsMemory(0, length));
         }
 
-        internal JsonClaimSet CreatePayloadClaimSet(ReadOnlySpan<byte> byteSpan)
+        internal JsonClaimSet CreatePayloadClaimSet(Memory<byte> tokenPayloadAsMemory)
         {
-            Utf8JsonReader reader = new(byteSpan);
+            Utf8JsonReader reader = new(tokenPayloadAsMemory.Span);
```

Comment:
do we need to check that `reader` is not null?

Reply:
Not here, no, I don't think.